### PR TITLE
CR-1208 | Pointing to the new release version of census-int-common-service

### PR DESCRIPTION
**Pull request by akritisinhaa**
The EstabType has been update in census-int-common-service and also OTHER type has been modified to fix an existing bug. A new version of census-int-common-service has been released which is 0.0.66. Updating pom.xml to point to this new version.

Also some tests have been updated as they were using the EstabTypes which do not exist anymore after the update to the EstabType enum.

For more details on the changes in census-int-common-service please have a look at https://collaborate2.ons.gov.uk/jira/browse/CR-1208.

### CODE REVIEW

File: src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplModifyCaseTest.java [hunk 1]
```diff
@@ -206,7 +206,7 @@ public void shouldModifyAddress_RequestHH_ExistingHouseHold() throws Exception {
 
   @Test
   public void shouldModifyAddress_RequestHH_ExistingCastleHH() throws Exception {
-    verifyModifyAddress(CaseType.HH, EstabType.HOUSEHOLD, EstabType.CASTLES.getCode());
+    verifyModifyAddress(CaseType.HH, EstabType.HOUSEHOLD, EstabType.RESIDENTIAL_BOAT.getCode());
```
Comment (rpcatling) [inline src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplModifyCaseTest.java:209]: method name is now misleading. please change it

File: src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplModifyCaseTest.java [hunk 2]
```diff
@@ -226,7 +226,7 @@ public void shouldModifyAddress_RequestSPG_ExistingHouseHold() throws Exception
 
   @Test
   public void shouldModifyAddress_RequestSPG_ExistingCastleHH() throws Exception {
-    verifyModifyAddress(CaseType.SPG, EstabType.EMBASSY, EstabType.CASTLES.getCode());
+    verifyModifyAddress(CaseType.SPG, EstabType.EMBASSY, EstabType.MILITARY_SFA.getCode());
```
Comment (rpcatling) [inline src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplModifyCaseTest.java:229]: method name is now misleading. please change it

**Review by rpcatling — CHANGES_REQUESTED** [review]
just a couple of minor naming changes requested.

**Review by rpcatling — APPROVED** [review]
All good

**Review by MartinHumphrey — APPROVED** [review]

**Pull request merged**